scripts/generate_all_jobs.py

```diff
@@ -58,6 +58,13 @@ def main(argv=sys.argv[1:]):
     print('')
 
     config = get_index(args.config_url)
+    if args.config_url.startswith('file:'):
+        print('WARNING: Local file system path used for ',
+              'configuration. Configuration will not be ',
+              'accessible to jobs during execution. Consider ',
+              'using a web(http) hosted configuration repository.',
+              file=sys.stderr)
```

Comment:
Just as a note - the current state can be kept as is:

> About the subsequent lines, this is the indentation enforced by both flake8 and pylint, so I don't know how to change it without failing more checks.

If you don't wrap right after the open parenthesis the following lines must be indented to the same level. Commonly if any arguments are wrapped we ensure to wrap *all* of them, so having a newline right after the open parenthesis. That way all arguments are indented exactly one level.

Reply:
Oh, now I understand what you mean. Thanks.
While I was already editing the code I now changed it as well.